cli_tools/import_precheck/check_disks_linux.go

```diff
@@ -46,6 +46,33 @@ func (c *disksCheck) getMBR(devName string) ([]byte, error) {
 	return data, nil
 }
 
+func (c *disksCheck) readMounts() (*mountPoints, error) {
+	var lsblkOut []byte
+	var err error
+	if c.lsblkOverride != nil {
+		lsblkOut, err = c.lsblkOverride()
+	} else {
+		cmd := exec.Command("lsblk", "--json", "--output", "name,mountpoint,type")
+		lsblkOut, err = cmd.Output()
+		if err != nil {
+			exitErr := err.(*exec.ExitError)
```

Comment:
Is it always ExitError?

Reply:
Probably not, good call; I copied that from the original code too quickly. 

Updated to allow for other types of errors.